docs/pages/index.md

```diff
@@ -23,7 +23,7 @@ display-title: "false"
         <div class="overview">Quick&nbsp;Start</div>
     </a>
     <ul><b>Getting Started</b>
-        <li><a href="{{ site.baseurl }}/overview">Introducing Meshery</a></li>
+        <li><a href="{{ site.baseurl }}/project">Introducing Meshery</a></li>
```

Comment:
@Anushka-shukla Could you change it to `/getting -started/overview`
Also, this is not the only file to be changed, do refer this comment https://github.com/layer5io/meshery/issues/3023#issuecomment-848069286

Reply:
done